pkgs/development/libraries/qt-6/modules/qtwebengine.nix

```diff
@@ -131,6 +131,8 @@ qtModule {
   patches = [
     ../patches/qtwebengine-darwin-no-low-latency-flag.patch
     ../patches/qtwebengine-darwin-no-copy-certificate-chain.patch
+    # Don't assume /usr/share/x11, see https://github.com/NixOS/nixpkgs/issues/226484
```

Comment:
nit: I'd prefer a comment like „allow dynamic override of xkeyboard-config location via XKB_CONFIG_ROOT for custom layouts“ to make it clearer that the previously done static override doesn't work (without having to open the issue)

Reply:
Line 130 should also be moved downward.

Reply:
Updated both of those, thanks!
You're right that the original comment wasn't very clear/precise.